JSDemos/Demos/TreeList/MultipleRowSelection/description.md

```diff
@@ -1 +1,7 @@
-In&nbsp;this demo, the TreeList component uses the multiple selection mode. To&nbsp;select a&nbsp;row, click the check box located in&nbsp;this row. Note that selection is&nbsp;non-recursive by&nbsp;default, that&nbsp;is, only the clicked row is&nbsp;selected, but not its nested rows. To&nbsp;make it&nbsp;recursive, click the _&laquo;Recursive Selection&raquo;_ check box under the TreeList. You can also select all rows at&nbsp;once by&nbsp;clicking the check box in&nbsp;the column header.
\ No newline at end of file
+This demo enables the *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to select multiple rows using checkboxes or keyboard shortcuts.
```

Comment:
1. Normally, we don't use articles with modes (just the way you did below).
2. I would have avoided "using" if possible.
```suggestion
This demo enables *"multiple"* **selection**.[mode](/Documentation/ApiReference/UI_Components/dxTreeList/Configuration/selection/#mode). This mode allows users to press keyboard shortcuts or click checkboxes to select multiple rows.
```